sdk/dotnet/Pulumi/Core/InputList.cs

```diff
@@ -55,19 +55,29 @@ private InputList(Output<ImmutableArray<T>> values)
         {
         }
 
-        public void Add(params Input<T>[] inputs)
```

Comment:
This never worked as intended; `params` does not avoid multiple invocations for each elements, so I removed it.